sysml.library/Kernel Library/Occurrences.kerml

```diff
@@ -161,19 +167,19 @@ package Occurrences {
 	}
 	
 	/**
-	 * TimeSliceOf asserts once occurrence is a time slice of another.
+	 * TimeSliceOf asserts once occurrence is a time slice of another, including at least itself.
 	 */
 	assoc TimeSliceOf {
```

Comment:
Same comment as above for time slices, they should specialize HappensDuring:
  assoc TimeSlicesOf specializes HappensDuring.
Same for another one noted below.  Can file a separate issue if you want.

Reply:
Shouldn't `TimeSliceOf` specialize `PortionOf`?